tests/test_singularity_containers.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+import spikeinterface.sorters as ss
+from spikeinterface.core.testing_tools import generate_recording
+
+recording = generate_recording(
+    num_channels=32,
+    sampling_frequency=30000.0,  # in Hz
+    durations=[120],
+    set_probe=True,
+    ndim=2,
+)
+
+recording2 = recording.save(folder="test_recording")
+
+kwargs = dict(recording=recording2, verbose=True, singularity_image=True)
+
+
+@pytest.mark.xfail(reason="FAILING: investigate why")
```

Comment:
This is happening because of inconsistent naming format:
- `spykingcircus` used to call run_sorter
- `spyking_circus` as python module
- `spyking-circus` as image name

I have created this [PR](https://github.com/SpikeInterface/spikeinterface/pull/714/files) to properly raise the error

Reply:
I changed `SORTER_DOCKER_MAP ` in `run_sorter.py` from spikeinterface for tests.
We need to unify naming format for spyking-cirucs

Reply:
Also I'm getting error using data from `generate_recording` for spyking-circus.

When I change to toy data it works well:
```
se.toy_example(
        duration=30,
        seed=0,
        num_channels=32,
        num_segments=1
    )
```

Reply:
The image runs well, but It was raising [here](https://github.com/SpikeInterface/spikeinterface/blob/b55ddc37345296bbe174aa7f28ff32c4a7815f68/spikeinterface/extractors/spykingcircusextractors.py#L54) when extracting output.
I could see different `result[...].hdf5` files in output folder, but not `'result-merged.hdf5'` or `'result.hdf5'`